README.md

```diff
@@ -38,21 +40,22 @@ Future features include managing/creating email campaigns and ticket exchanges.
+   2. Set values for the `PRIVATE_STRIPE_KEY` and `PUBLIC_STRIPE_KEY`. For testing, we also send requests from locally running instances of Wondertix to an external Stripe account. The Stripe account we use for this is `wtix-dev`. Likewise, if you don't already, ask a team lead for access and/or for the values of `wtix-dev`'s private and public keys.
 3. Create mkcert certificates.
    1. Navigate to `<path/to/WonderTix/server>` and `<path/to/WonderTix/client>`.
    2. Run `mkcert -install` to install the local certificate authority.
-   3. Run `mkcert localhost` to create a certificate.   
-4. Run `docker-compose up -d` (or `docker compose up` to see live container logs in the terminal). This starts a locally running instance of Wondertix using docker containers. 
-5. To test the checkout process with Stripe, make sure you have the [Stripe CLI](https://stripe.com/docs/stripe-cli) installed. 
-   1. Once the Stripe CLI is installed, run `stripe login` and press enter to open the browser. 
-   2. In the browser that the Stripe CLI opens, make sure you select `Wondertix Dev` as the account, verify that the code matches that in your terminal, and select `Allow Access`. If `Wondertix Dev` does not appear as an option, you will need to ask for access to the account. You should only need to login to the account like this once. 
-   3. Back in your terminal, run `stripe listen --forward-to https://localhost:8000/api/2/order/webhook --events checkout.session.completed,checkout.session.expired` and copy the resulting ***signing secret*** as your `PRIVATE_STRIPE_WEBHOOK` variable in the `.env` file. This tells Stripe to forward events to your local webhook. See the `stripe listen` command [API docs](https://stripe.com/docs/cli/listen) for more details. 
-      - __NOTE__ - stripe listen _must_ remain running in your terminal for events to be forwarded and, subsequently, for the checkout process to complete. This means that everytime you wish to test the checkout process locally, you have to run the `stripe listen` command and leave it running. You should see your device listed in the Stripe developer portal on the `Wondertix Dev` account under `Webhooks -> Local listeners` and the `Status` should be `Listening`. 
-   4. You should now be able to sucessfully checkout with Stripe locally and see order details updated. See [Test Payment Methods](https://stripe.com/docs/testing) for how to checkout with Stripe in test mode. 
-6. The client will be available at <https://localhost:3000>
-7. The server will be available at <https://localhost:8000>
-8. The swagger docs will be available at <https://localhost:8000/api/docs>
+   3. Run `mkcert localhost` to create a certificate.
+4. Get the google cloud storage service account key file from your team lead and put into `<path/to/WonderTix/server>` for access to image upload.
```

Comment:
Related to my earlier comment on `.env.dist`, configuring the service account key like this means that we would have no way to provide it to deployed environments without trying to do something fancier like mounting the file onto cloudrun which, like I mentioned, am not sure is possible and is not really necessary since this could just be another environment variable.

All environment variables used by deployed environments (dev,stg,prd) are stored in GCP's secret manager which hides the actual value. So, if the only way to get image uploads to work locally with an environment variable is to commit the secret value to `.env.dist`, then we could just not do that and image upload would only work for deployed environments once we add add the secret to secret manager.

If someone really wants to test out image upload locally, then they could do very similar to what is mentioned here already which would be to reach out to the TL and add the secret to there own `.env`.

Feel free to reach out if you have questions about what I'm trying to get at

Reply:
Talked with @ReidLuttrell and the proposed solution is to save the key file as a string in the gcp secret for deployed environments. Verified this works locally